content/departments/product-engineering/engineering/cloud/delivery/managed/oidc_site_admin.md

```diff
@@ -0,0 +1,103 @@
+# Purpose and Background
```

Comment:
Can you add a section that mentions this approach requires an increased seat count (I think we decided on 10)? We'll need to make sure CE is aware so they can adjust the license appropriately.